app/src/main/java/org/mozilla/fenix/theme/FirefoxTheme.kt

```diff
@@ -107,9 +105,7 @@ private val lightColorPalette = FirefoxColors(
     layer3 = PhotonColors.LightGrey20,
     layerAccent = PhotonColors.Ink20,
     layerNonOpaque = PhotonColors.Violet70A12,
-    scrim = PhotonColors.DarkGrey05A45,
-    scrimAccentStart = PhotonColors.DarkGrey90A96,
-    scrimAccentEnd = PhotonColors.DarkGrey30A96,
```

Comment:
We are intentionally removing the scrim gradient to use a single scrim background color.